pkg/util/uuid/uuid.go

```diff
@@ -0,0 +1,24 @@
+package uuid
+
+import (
+	"github.com/gofrs/uuid"
+)
+
+// This is a facade to aid in conversion from github.com/satori/go.uuid to github.com/gofrs/uuid
+// Changing due to https://www.whitesourcesoftware.com/vulnerability-database/WS-2018-0594
+
+// helper so we can in-line uuid generation with gofrs, drops the error that would be returned
+func NewV4() uuid.UUID {
+	u, _ := uuid.NewV4()
```

Comment:
must read the error and panic if it is set; not to do so here could be a security risk